### ISSUE TITLE
Invalid default map group

### ISSUE THREAD

**Issue by jenrik** [ShootingRange/csgo-container#1]
The default map group, `mp_active`, is either misspelled or otherwise incorrect.

Error message emitted by CS:GO:
```
Setting mapgroup to 'mp_active'
ApplyGameSettings: Invalid mapgroup name mp_active
```

**Comment by jenrik** [comment]
The map group is misspelled and should be `mg_active`

**Comment by jenrik** [comment]
fixed in f232af0d97dd0c50477b17dfcdab87df21200a82
